schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
+CREATE DATABASE vet_clinic;
+
 CREATE TABLE animals (
-    name varchar(100)
+    id int,
```

Comment:
- [ ]  **[OPTIONAL]** It would be nice to mark the `id` column as the `primary key` and also make the `id` column `not null` and `autoincremented` so that you don't have to keep track of the next id your data should have. This will be taken care of automatically for you. See suggestion below;
```suggestion
    id int NOT NULL PRIMARY KEY GENERATED ALWAYS AS IDENTITY
```